index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <h1 id="title">
+        freeCodeCamp Survey Form
+        </h1>
+        
+        <p id="description">
+          Thank you for taking the time to help us improve the platform
+        </p>
+        
+        <form id="survey-form">
```

Comment:
The form does submit the form details using the get method. 
       - **Why**
       You should consider using the `POST` method to ensure that the results are more secure and not exposed in the URL link.
       
![Screenshot from 2023-01-17 11-59-45](https://user-images.githubusercontent.com/65118995/212853983-6e33eb04-1d35-4617-baf1-a19bca3fc33a.png)